spring-integration-core/src/test/java/org/springframework/integration/dsl/routers/RouterTests.java

```diff
@@ -936,15 +952,19 @@ public IntegrationFlow propagateErrorFromGatherer(TaskExecutor taskExecutor) {
 					.get();
 		}
 
+		@Bean
+		public PollableChannel scatterGatherWiretapChannel() {
```

Comment:
I would prefer `scatterGatherWireTapChannel`. I mean, please, ensure every word in the name is came-cased. 
Much easier to read.

Reply:
I realized that in framework everywhere, wireTap, T is capital. I have fixed this as well.